fdmi/plugins/fdmi_app_word_count

```diff
@@ -0,0 +1,427 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# This is the FDMI sample application. It serves as a hub for the FDMI
+# records delivered fdmi_sample_plugin C-program via stdout. To do so
+# it gathers the cluster configuration with 'hctl status' command,
+# forks fdmi_sample_plugin and waits for the FDMI events in JSON form.
+# The reason to use it is simply because it is much easier to parse data
+# and develop new features in python in compare to C.
+
+# For now it is just a demo that prints FDMI records to console, but
+# except this it can perform records de-dup and for example, replication
+# coordination by means of set of the replication python programs that
+# perform actual data move.
+
+import subprocess
+import signal
+import shlex
+import sys
+import json
+import codecs
+import requests
+import re
+import boto3
+import string
+from time import sleep
+import os
+
+# Global cluster info including information required for running
+# fdmi_sample_plugin C-program.
+cluster_info = dict()
+
+# Global key-value cache used for records de-duplication. KV pairs
+# are available by the kv['cr_key']
+#kv_records = dict()
+kv_records = []
+
+# Global list of fdmi_sample_plugin processes that this program
+# is communicating.
+processes = []
+
+# Global word_counts dictionary
+word_counts = dict()
+
+def make_request(kv):
```

Comment:
This is not used. Please remove it.

Reply:
This is actually used for storing the count of the words inside the object data to be able to print to the console.

Reply:
I didn't see it is used anywhere in this patch.

Can you explain where this function `make_request()` is used?
And remove those unused code in this function.

Reply:
Hi @huanghua78 the make_request() function is just to give an option for the case of communicating with another server through HTTP to send the filtered FDMI record. It is not used for the fdmi_app_word_count example I am adding here. I just keep it there for potential reference to other FDMI application examples.